include/swift/SILOptimizer/Utils/InstOptUtils.h

```diff
@@ -284,14 +284,14 @@ void releasePartialApplyCapturedArg(
 /// Insert destroys of captured arguments of partial_apply [stack].
 void insertDestroyOfCapturedArguments(
     PartialApplyInst *pai, SILBuilder &builder,
-    llvm::function_ref<bool(SILValue)> shouldInsertDestroy =
-        [](SILValue arg) -> bool { return true; },
+    llvm::function_ref<SILValue(SILValue)> getValueToDestroy =
```

Comment:
In a subsequent commit, can you add to this comment a small blurb/example showing how to use getValueToDestroy. I think it would make sense to make it clear that you can use it to either change the value to destroy /or/ to return SILValue() if you do not want any destroy inserted. Otherwise, I am worried that people will read the API and think the SILValue returned needs to always be valid.